samples/embeddings/csharp-inproc/local.settings.json

```diff
@@ -2,6 +2,8 @@
   "IsEncrypted": false,
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "AZURE_OPENAI_ENDPOINT": "https://<resource-name>.openai.azure.com/",
+    "EMBEDDING_MODEL_DEPLOYMENT_NAME": "text-embedding-ada-002"
```

Comment:
Should we standardize on `text-embedding-3-small`? This is now the preferred model by OpenAI to replace `text-embedding-ada-002` since it's faster and cheaper. It will be hard to change this default later, so I'm wondering if we should do it now (assuming that Azure OpenAI supports it too). More info here: https://platform.openai.com/docs/guides/embeddings/embedding-models